digital-literacy/computer-basics.md

```diff
@@ -45,95 +52,116 @@ Click, then drag the mouse over the area of the page you want to highlight to se
 Depending on how old your computer is, there will be different kinds of ports along the side of it. These include headphone jacks, charging ports and USB ports for external harddrives and thumb drives. These ports allow you to connect your computer to other pieces of hardware to expand what you can do on your computer. Some common uses for the ports on your computer are connecting it to chargers to charge your computer, large screens or monitors to enlarge the screen that you’re working on, and harddrives to save materials on a backup disk. Sometimes your computer requires an adapter (an additional kind of wire) to connect another piece of hardware to the computer if the port on your computer is different from the connecting wire on the piece of hardware.
 
 ### Using the Keyboard
+
 An overview of the Mac keyboard can be found [here](https://keyshorts.com/blogs/blog/41999105-the-ultimate-guide-to-macbook-keyboard).
 
 An overview of a Windows/PC keyboard can be found [here](https://support.microsoft.com/en-us/windows/using-your-keyboard-18b2efc1-9e32-ba5a-0896-676f9f3b994f).
 
 Make sure your mentee feels comfortable with the basics of typing: letters, capitalization, caps lock, numbers, and special characters with the shift key. Computer controls such as volume and brightness should also be accessible. Additionally, you can show your mention how to type special characters with the alt/option + shift combination.
 
 ### Files & Folders
+
 #### Creating Folders
+
 \*This is Mac specific
```

Comment:
Not related to your change, but since you're editing this file, we should just bold this line and add a period to end so it renders correctly.
```suggestion
**This is Mac specific.**
```